src/common/transformations/src/transformations/op_conversions/convert_pad_to_group_conv.cpp

```diff
@@ -57,6 +58,15 @@ ov::pass::ConvertPadToGroupConvolution::ConvertPadToGroupConvolution() {
             return false;
         }
 
+        // check that Pad has non-negative values
```

Comment:
Are these checks absolutely the same as in AvgPool, Convolution Fusion transformations?
Maybe it's better to move them in one place